pkg/distributors/contracts/MultiRewards.sol

```diff
@@ -367,12 +374,10 @@ contract MultiRewards is IMultiRewards, IDistributor, ReentrancyGuard, Temporari
         IERC20[] calldata pools,
         address callbackContract,
         bytes calldata callbackData
-    ) public nonReentrant {
+    ) public nonReentrant returns (bytes memory) {
         _getReward(pools, payable(callbackContract), true);
 
-        (bool success, ) = callbackContract.call(callbackData);
-        // solhint-disable-previous-line avoid-low-level-calls
-        require(success, "callback failed");
+        return Address.functionCall(callbackContract, callbackData);
```

Comment:
This is a sidenote to the Exit contract but doesn't this allow anyone to drain MultiRewards? I can just encode a transfer of all of the contract's assets to my own address. We need to have some controls on here.